Dockerfile.sdk

```diff
@@ -38,6 +38,8 @@ ARG TRITON_ENABLE_GPU=ON
 
 FROM ${BASE_IMAGE}
 
+ARG TARGETPLATFORM
```

Comment:
TARGET_PLATFORM to make it consistent with the way others ARGs are named?

Reply:
This variable comes from "docker buildx" so we don't control it name.